hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/IMapSqlConnector.java

```diff
@@ -155,31 +185,44 @@ public Vertex fullScanReader(
                 filter
         );
 
-        return dag.newUniqueVertex(table.toString(), OnHeapMapScanP.onHeapMapScanP(mapScanMetadata));
+        return dag.newUniqueVertex(table.getMapName(), OnHeapMapScanP.onHeapMapScanP(mapScanMetadata));
```

Comment:
What's wrong with `table.toString()`?

Reply:
More clear description in debug info; easier to test.

Reply:
Maybe `toString()` defined in this class is what you need? (Consider adding it to `PartitionedMapTable`)